### PR TITLE
Update dependencies - closes #36

### LINKED ISSUE #36: Bump dependencies
We have to update some dependencies like puppeteer.

### PR DESCRIPTION
**Pull request by robincher**
Mainly to update node dependencies for
1) kleur
2)puppeteer

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "website-checks",
-  "version": "1.17.0",
+  "version": "1.18.0",
```
Comment (DanielRuf) [inline package.json]: Please revert this =) This is automatically done by `yarn version` by me.

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,762 @@
+{
```
Comment (DanielRuf) [inline package-lock.json]: Please remove this lockfile. We use yarn and lockfiles are generated and updated by me.

File: .gitignore [hunk 3]
```diff
@@ -1,2 +1,7 @@
+
+#Libraries and modules
 node_modules
-.DS_Store
\ No newline at end of file
+.DS_Store
+
+# Test output files
+iana.org
```
Comment (DanielRuf) [inline .gitignore]: Please make a separate branch and PR for the gitignore changes.

**Comment by DanielRuf** [comment]
Hi @robincher,

thank you very much for your contribution and PR.
I have provided some feedback on your proposed changes.

Please also use the following commit format:
> Bump node dependencies - closes #36

So the automated kanban works with this and the issue is automatically closed when this commit is merged.

File: .gitignore (hunk 3 above)
Comment (robincher) [reply .gitignore]: Noted :)

**Comment by robincher** [comment]
Let me update this PR after this weekend or so.

**Comment by DanielRuf** [comment]
Hi @robincher,

thanks for your contribution =)
Please create a separate branch from the master branch next time =)

**Comment by kkrieger85** [comment]
@DanielRuf You can merge this PR now.

**Comment by DanielRuf** [comment]
Thanks for the review. I'll merge it later.

**Comment by DanielRuf** [comment]
Hey @robincher,

thanks for your contriobution to website-checks.
I have sent you an invitation as collaborator with a few more rights.

You can accept it at https://github.com/DanielRuf/website-checks/invitations

**Comment by DanielRuf** [comment]
Hey @robincher,

thanks for your contribution to website-checks.
I have sent you an invitation as collaborator with a few more rights.

You can accept it at https://github.com/DanielRuf/website-checks/invitations

**Pull request merged**